src/broker/overlay.c

```diff
@@ -1974,25 +1974,25 @@ static const struct flux_msg_handler_spec htab[] = {
         FLUX_MSGTYPE_REQUEST,
         "overlay.stats.get",
         overlay_stats_get_cb,
-        0
+        FLUX_ROLE_USER,
     },
     {
         FLUX_MSGTYPE_REQUEST,
         "overlay.health",
         overlay_health_cb,
-        0
+        FLUX_ROLE_USER,
     },
     {
         FLUX_MSGTYPE_REQUEST,
         "overlay.disconnect",
         disconnect_cb,
-        0
+        FLUX_ROLE_USER,
```

Comment:
Am I reading this wrong, or does this allow a user to force a broker disconnect?

Reply:
No,  that is for cleaning up `flux overlay status --wait=STATE` clients that disconnect before STATE is reached.

The RPC topic for `flux overlay disconnect` is `overlay.disconnect-subtree`.

I'll add a comment b/c that is confusing!

Reply:
Ah! That makes sense now (I forgot momentarily about the `*.disconnect` handlers)